### PR TITLE
[Chips] add missing theming extension documentation.

### PR DESCRIPTION
**Pull request by fluxminds**

### CODE REVIEW

File: components/Chips/README.md [hunk 1]
```diff
@@ -21,12 +21,45 @@ Chips are compact elements that represent an input, attribute, or action.
...
+  <li class="icon-list-item icon-list-item--link">Protocol: <a href="https://material.io/components/ios/catalog/chips/api-docs/Protocols/MDCTextInputPositioningDelegate.html">MDCTextInputPositioningDelegate</a></li>
   <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/chips/api-docs/Enums.html">Enumerations</a></li>
   <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/chips/api-docs/Enums/MDCChipFieldDelimiter.html">MDCChipFieldDelimiter</a></li>
+  <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/chips/api-docs/Enums/MDCCornerTreatmentValueType.html">MDCCornerTreatmentValueType</a></li>
+  <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/chips/api-docs/Enums/MDCTextInputTextInsetsMode.html">MDCTextInputTextInsetsMode</a></li>
```
Comment (jverkoey) [inline components/Chips/README.md]: This output is a bug. Will file an issue to track and resolve.

Comment (jverkoey) [reply components/Chips/README.md]: https://github.com/material-components/material-components-ios/issues/7195

Comment (jverkoey) [reply components/Chips/README.md]: Please revert these changes for now; we can re-run the generator later to make sure we're adding any missing APIs, but as this presently stands this output is a bug.

Comment (fluxminds) [reply components/Chips/README.md]: Addressed in 521bbd19fcc67dbc89a4d5e453ae5f20a459a52a

File: components/Chips/docs/theming-extension.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+### Theming Extensions
```
Comment (jverkoey) [inline components/Chips/docs/theming-extension.md]: Rename this file and it’s header to “Theming”.

File: components/Chips/docs/theming-extension.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ ...
+ // Create or use your app's Container Scheme
+MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
+ // Theme the chip with either default theme
+[self.chip applyThemeWithScheme:containerScheme];
+ // Or outlined theme
+[self.chip applyOutlinedThemeWithScheme:containerScheme];
+```
+
+<!--</div>-->
```
Comment (jverkoey) [inline components/Chips/docs/theming-extension.md]: Please add this file to the docs/README.md table of contents and re-run the readme generator.

Comment (fluxminds) [reply components/Chips/docs/theming-extension.md]: Addressed in 1823e91 and e0437b0.

File: components/Chips/docs/theming-extension.md (hunk 2 above)
Comment (fluxminds) [reply components/Chips/docs/theming-extension.md]: Addressed in 1823e91 and e0437b0.

**Review by jverkoey — APPROVED** [review]

**Pull request merged**